packages/flutter/lib/src/services/message_codec.dart

```diff
@@ -187,6 +187,10 @@ class PlatformException implements Exception {
   /// ```
   ///
   /// In the cases described above, the content of [stacktrace] will be the unprocessed output of calling `toString()` on the exception.
+  ///
+  /// macOS, iOS, Linux and Windows don't support querying the native stacktrace.
```

Comment:
nit: sentences should start grammatically correct with a capital letter, see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar

Reply:
Done in b057d42c43e27961bd656cb1384dc527a2878de1